packages/api-key/src/services/api-key-module-service.ts

```diff
@@ -103,36 +142,257 @@ export default class ApiKeyModuleService<TEntity extends ApiKey = ApiKey>
     data: ApiKeyTypes.UpdateApiKeyDTO[] | ApiKeyTypes.UpdateApiKeyDTO,
     @MedusaContext() sharedContext: Context = {}
   ): Promise<ApiKeyTypes.ApiKeyDTO[] | ApiKeyTypes.ApiKeyDTO> {
```

Comment:
q: looks like the DTO does not expect the salt to be returned, should we just remove it entirely? I join riqwan on that. Otherwise it feels weird that the salt is present but not expected to be, you might wonder if something wrong happen?

Reply:
It was convenient in the test, but I agree on deleting at least the salt 👍